eslint.config.mjs

```diff
@@ -58,6 +58,15 @@ const c = tseslint.config(
   {
     files: ['**/*.json', '**/*.[tj]s?(x)', '**/*.cjs', '**/*.mjs'],
   },
+  // Node environment
+  {
+    files: ['jest/**/*.[js|mjs]', '**/*/babel.config.js', '**/*/rollup.config.mjs', '.rollup/index.mjs'],
```

Comment:
Not sure the best way to target these. I think maybe we could assume all `[js|mjs]` are node environment then override the `globals` later to remove node for certain browser-only packages etc.